File: POO/Polimorfismo/src/br/escola/Funcionario.java
package br.escola;

public class Funcionario extends Pessoa {
    private double salario;
    
    public Funcionario(String nome, String email, double salario){
        super(nome, email);
        this.salario = salario;
    }
    
    @Override
    public void mostrarInfo(){
        super.mostrarInfo();
        System.out.println("Salario: " + this.salario);
    }
}

File: PI-III-B/2023-07-07/Funcionario.php
<?php 

//Criando a classe Funcionário, herdando de Pessoa.
class Funcionario extends Pessoa {
   
  //Criando os atributos.
    public $cargo;
    private $salario;
    public $dataIngresso;
  
    // Método para exibir os dados do funcionário
    public function exibir_dados() {
      echo "Nome: " . $this->nome . "<br>";
      echo "CPF: " . $this->cpf . "<br>";
      echo "Data de Nascimento: " . $this->nascimento . "<br>";
      echo "Cargo: " . $this->cargo . "<br>";
      echo "Salário: " . $this->salario . "<br>";
      echo "Data de Ingresso: " . $this->dataIngresso . "<br>";
  }

}

?>